cfgov/v1/jinja2/v1/blog/legacy_blog_page.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'v1/layouts/layout-2-1.html' %}
```

Comment:
Instead of inheriting this from layout-2-1, would it make more sense to extend from blog_page.html, and only overwrite the `content_intro` and `content_main` blocks? This would avoid having to redefine all of the other blocks that just duplicate what's in blog_page.

Reply:
See https://github.com/cfpb/consumerfinance.gov/pull/8484/commits/8b86d23a2e913ed2b0a2c655a5b86b9b9e339a16!